homeassistant/components/evohome.py

```diff
@@ -0,0 +1,613 @@
+"""Support for Honeywell evohome (EMEA/EU-based systems only).
+
+Support for a temperature control system (TCS, controller) with 0+ heating
+zones (e.g. TRVs, relays) and, optionally, a DHW controller.
+
+A minimal configuration.yaml is as as below:
+
+evohome:
+  username: !secret evohome_username
+  password: !secret evohome_password
+
+These config parameters are presented with their default values:
+
+# scan_interval: 180     # seconds, you can probably get away with 60
```

Comment:
We shouldn't expose scan interval. We should pick one that makes sense and stick with it.

Reply:
I think that the existing Honeywell component has a configurable scanning table?

Anyway I'm happy to do this. Accepted wisdom might say 300 seconds but would you be OK if I did 180 seconds?

This would be well below the API rate limit.

Reply:
OK, set to 180 seconds.  I left the backoff in, as api rate can be exceeded for other reasons.